libs/hooks/open-telemetry/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@openfeature/open-telemetry-hook",
-  "version": "6.0.1",
+  "name": "@openfeature/open-telemetry-hooks",
+  "version": "0.1.0",
```

Comment:
As discussed in the linked issue, we will be deprecating the old package and putting both OTel hooks in this new one.